.github/workflows/pr.yml

```diff
@@ -18,16 +18,3 @@ jobs:
       steps:
         - name: Checkout Code
           uses: actions/checkout@v2
-
-        # Initializes the CodeQL tools for scanning.
```

Comment:
Was CodeQL not being used for anything? Wanted to check before we remove it.

Reply:
Yup! It has never done anything with the analysis, and I've been meaning to remove it. Didn't remember removing it honestly haha.